library/src/main/kotlin/com/chuckerteam/chucker/internal/support/RequestProcessor.kt

```diff
@@ -31,6 +31,9 @@ internal class RequestProcessor(
                 setGraphQlOperationName(it)
             }
             populateUrl(request.url)
+            isGraphQLRequest = this.graphQlOperationName != null ||
+                request.url.pathSegments.contains("graphql") ||
+                request.url.host.contains("graphql")
```

Comment:
Let's move this to a separate function if possible.

Reply:
done